### PR TITLE
Fixes generation of minified assets and allows the gui to serve minified code via config.

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2795/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2796/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2800/
Test PASSed.

File: jujugui/static/gui/src/app/assets/javascripts/unscaled-pack.js [hunk 1]
```diff
@@ -10,9 +10,9 @@
 * not based on circle packing.  For now, this remains a patch.
 */
 
-import "../../../node_modules/d3/src/layout/layout";
-import "../../../node_modules/d3/src/layout/hierarchy";
-import "../../../node_modules/d3/src/layout/tree";
+import "../../../../../../../node_modules/d3/src/layout/layout";
```
Comment (makyo) [inline jujugui/static/gui/src/app/assets/javascripts/unscaled-pack.js]: Gross, but necessary :/

**Comment by makyo** [comment]
Code :+1: - no QA yet

**Comment by hatched** [comment]
:+1:

**Comment by hatched** [comment]
QA OK!

**Comment by jaycee** [comment]
Thanks @hatched and @makyo.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**